examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -119,10 +137,9 @@ typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
-    current<UsedParticleCurrentSolver>
-    #if(PARAM_IONIZATION==1)
-    ,ionizer<particles::ionization::BSI<PIC_Electrons> >
-    #endif
+    current<UsedParticleCurrentSolver>, 
+    atomicNumbers<Helium>,
+    ionizer<particles::ionization::BSI<PIC_Electrons> >
```

Comment:
yes, what I meant with that was: we do not need a "general" `ENABLE_IONIZATION` flag since we will just add something like `ionizer<particles::ionization::BSI<PIC_Electrons> >` (as it is now).

therefore you just need a `PARAM_IONIZATION` in the *single example* that is modified to enable/disable the *test*.

in short: there should be a `PARAM_IONIZATION` used in `examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param` but none in the other files. also, please use the `#if(defined ...)` statements I showed you earlier.